### PR TITLE
Issue #3019: Don't display install profile version if empty.

### LINKED ISSUE #3019: Support install profiles with no version number
I'm using a custom install profile that doesn't have a version number. When I go to admin/reports/status it says:

> Install profile: PackWeb (pw_profile-)

The hyphen at the end is meant to separate the install profile name from it's version, but since there's no version number you just get that.

It's a very minor issue, but can we make it not add the hyphen if there's no version number?
For reference, this comes from [line 35 of system.install](https://github.com/backdrop/backdrop/blob/1.x/core/modules/system/system.install#L35).

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/backdrop/backdrop-issues/issues/3019

### CODE REVIEW

File: core/modules/system/system.install [hunk 1]
```diff
@@ -27,13 +27,22 @@ function system_requirements($phase) {
     $profile = backdrop_get_profile();
     if ($profile != 'standard') {
       $info = system_get_info('module', $profile);
-      $requirements['install_profile'] = array(
-        'title' => $t('Install profile'),
-        'value' => $t('%profile_name (%profile-%version)', array(
+      if (!empty($info['version'])) {
+        $value = $t('%profile_name (%profile-%version)', array(
```
Comment (klonos) [inline core/modules/system/system.install]: Simpler? 🙂

```php
      // If a version has been specified, append it to the profile.
      $profile .= !empty($info['version']) ? '-' . $info['version'] : '';
      $requirements['install_profile'] = array(
        'title' => $t('Install profile'),
        'value' => $t('%profile_name (%profile)', array('%profile_name' => $info['name'], '%profile' => $profile)),
        'severity' => REQUIREMENT_INFO,
        'weight' => -9
      );
```

Comment (ghost) [reply core/modules/system/system.install]: You're too clever for me :stuck_out_tongue: 
PR updated.

**Review by klonos — APPROVED** [review]
👍

**Review by klonos — APPROVED** [review]
LGTM 👍

**Comment by quicksketch** [comment]
Merged into 1.x and 1.18.x.

**Pull request merged**